src/engine.ts

```diff
@@ -572,6 +581,29 @@ export default class Engine {
 
   }
 
+  possibleSpecialActions(player: PlayerEnum) {
+    const commands = [];
+    const specialacts = [];
+
+    for (const event of this.player(player).events[Operator.Activate]) {
```

Comment:
Should check if the event isn't already activated

Reply:
thanks, right!